app/assets/stylesheets/variables/_colors.scss

```diff
@@ -1,5 +1,6 @@
 $aqua: #7fdbff !default;
-$blue: #0071bc !default;
+$blue: #0071bb !default;
+$blue-medium: #cedced !default;
```

Comment:
Our `$border-color` is `#cedbec` which is super close to this so I think we can probably do without this variation.